src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/MD5.cs

```diff
@@ -35,8 +34,10 @@ protected MD5()
             HashSizeValue = HashSizeInBits;
         }
 
+        [UnsupportedOSPlatform("browser")]
         public static new MD5 Create() => new Implementation();
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
To match (e.g.) RSA and DSA, this one (the newly Obsoleted `Create(string)`) should be left out (I had skipped it in the proposal).